uber/templates/registration/form.html

```diff
@@ -271,6 +271,8 @@
                 {{ macros.checkbox(attendee, 'got_merch', label='Yes') }}
             </div>
         </div>
+    {% else %}
+        <input type="hidden" name="got_merch" value="{{ '1' if attendee.got_merch else '0' }}" />
```

Comment:
What was the reason for doing it this way rather than simply printing `{{ attendee.got_merch }}`? Did that not work?